auacm/app/util.py

```diff
@@ -39,3 +41,13 @@ def serve_response(response, response_code=200):
 def serve_error(error, response_code):
     '''Serve an error in response to a request.'''
     return jsonify({'status': response_code, 'error': error}), response_code
+
+def admin_required(function):
+    @wraps(function)
+    def wrap(*args, **kwargs):
+        if current_user.is_anonymous or current_user.admin == 0:
+            return serve_error('You need to be an admin to do that.',
```

Comment:
I wouldn't mind this being a little bit more specific. I'm not sure, but maybe the decorator could take an argument with a string describing the action, so `@admin_required` would be `@admin_required('create a problem')`. Probably not worth fixing in this PR